lib/reducers/create-otp-reducer.js

```diff
@@ -252,6 +252,7 @@ function createOtpReducer (config) {
   // validate the initial state
   validateInitialState(initialState)
 
+  // eslint-disable-next-line complexity
```

Comment:
Someday I would love to refactor all reducers using massive switch functions into maps of functions. That might just have to happen when we start using TypeScript in order to minimize the amount of expected formats the action argument can assume. So, this eslint-disable can be put here for now.

Reply:
Yea, probably something to keep in mind for the TypeScript refactor.